### ISSUE TITLE
npx create-instantsearch-app not working

### ISSUE THREAD

**Issue by zeke** [algolia/create-instantsearch-app#364]
Hi folks! 👋 

I ran this `npx` command found at https://www.algolia.com/doc/guides/building-search-ui/getting-started/js/?language=js#bootstrap-your-application, but it didn't seem to work. Can't tell if that last message is an error, but I don't see a new directory where I ran the command:

```
$ npx create-instantsearch-app ais-ecommerce-demo-app \
  --template "InstantSearch.js" \
  --app-id B1G2GM9NG0 \
  --api-key aadef574be1f9252bb48d4ea09b5cfe5 \
  --index-name demo_ecommerce \
  --attributes-to-display name

npx: installed 148 in 4.223s

Creating a new InstantSearch app in ais-ecommerce-demo-app.

The template directory must contain a configuration file `.template.js` or must be one of those: Angular InstantSearch, Autocomplete.js, InstantSearch Android, InstantSearch iOS, InstantSearch.js, InstantSearch.js 2, JavaScript Client, JavaScript Helper, React InstantSearch, React InstantSearch Native, Vue InstantSearch
```

**Comment by francoischalifour** [comment]
Thank you for filing an issue.

#365 should fix this.

**Comment by francoischalifour** [comment]
Released in [4.3.2](https://github.com/algolia/create-instantsearch-app/commit/81d5e9c7440aab253b3b152826a6f1061cb55812).

**Comment by zeke** [comment]
That was quick. Thanks!
